datastore/importers/nmdb-import-rpm-query/nmdb-import-rpm-query.cpp

```diff
@@ -0,0 +1,91 @@
+// =============================================================================
+// Copyright 2022 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS). Under the terms of Contract DE-NA0003525 with NTESS, the U.S.
+// Government retains certain rights in this software.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+// =============================================================================
+// Maintained by Sandia National Laboratories <Netmeld@sandia.gov>
+// =============================================================================
+
+#include <netmeld/datastore/tools/AbstractImportTool.hpp>
+#include "Parser.hpp"
+
+namespace nmdt = netmeld::datastore::tools;
+
+// =============================================================================
+// Import tool definition
+// =============================================================================
+template<typename P, typename R>
+class Tool : public nmdt::AbstractImportTool<P,R>
+{
+  // ===========================================================================
+  // Variables
+  // ===========================================================================
+  private:
+  protected:
+  public:
+
+  // ===========================================================================
+  // Constructors
+  // ===========================================================================
+  private:
+  protected:
+  public:
+    Tool() : nmdt::AbstractImportTool<P,R>
+      (
+       "rpm -qa",  // command line tool imports data from
```

Comment:
The import tool won't work unless a specific command is given for a specific output format.  The CLI line needs to be more explicit in that sense.  Something like:
```suggestion
       R"(rpm -qa --queryformat="%-50{NAME}%10{VERSION}%-%-20{RELEASE}%-20{ARCH}%{SUMMARY}\n")",  // command line tool imports data from
```